Firestore/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Unreleased
-- [Fixed] Remove explicit MobileCoreServices library linkage from podspec. (#6850)
+- [fixed] Remove explicit MobileCoreServices library linkage from podspec. (#6850)
```

Comment:
As long as you're in here, could you move the period to the end of the sentence (i.e. after the closing paren)?

Reply:
done.